docs/source/onboarding_resources/computing/index.rst

```diff
@@ -194,7 +243,7 @@ The cluster is accessed through the SSH ("Secure Shell") protocol, so what is ne
 
 PuTTY and Bitvise are both SSH clients with graphical user interfaces for Windows. For Mac, it looks like some exist (e.g. Termius) but I didn't see anything super popular.
 
-We should note that you can also just run ssh commands from a terminal without installing anything instead of using a graphical client. I do this, and I am guessing that Nathaniel does too.
+We should note that you can also just run ssh commands from a terminal without installing anything instead of using a graphical client. I do this, and I am guessing that Nathaniel does too. Include the link to the IHME training video on how to do this. 
```

Comment:
Maybe there is an edit to this line that will make it easier for future readers to understand.  Is Syl the speaker referred to as "I" in this case?

Reply:
I think I am the "I" here 😄

Reply:
Yes, I just `ssh` in from the terminal (technically I [mosh](https://mosh.org/) in), without using a GUI.

Reply:
Haha sorry - this should probably be in a "note" but some sections of this doc are mainly in note format. I am tackling one section at a time to go over, add, and clarify information. Cluster will be my next section so stay tuned for the next PR! 

And yes - Zeb is the "I" here! It is a copy paste of a comment he left so I didn't lose track of it.